linux_os/guide/services/ntp/ensure_chrony_is_configured/oval/shared.xml

```diff
@@ -0,0 +1,20 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("A remote NTP Server for time synchronization should be specified (and dependencies are met)") }}}
+    <criteria comment="chrony.conf conditions are met">
+      <criterion test_ref="test_chrony_remote_server_configuration" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="at_least_one_exists"
+  comment="Ensure at least one remote NTP server is set" id="test_chrony_remote_server_configuration" version="1">
+    <ind:object object_ref="object_chrony_remote_server_configuartion" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object comment="Ensure at least one remote NTP server is set"
+  id="object_chrony_remote_server_configuartion" version="1">
+    <ind:filepath operation="pattern match">^/etc/chrony.conf$</ind:filepath>
+    <ind:pattern operation="pattern match">^(\bserver[[:space:]]|pool[[:space:]]\b)([0-3]\b(.fedora|.suse|.rhel)\b\b(.pool.ntp.org)\b)|([0-3]\b(.pool.ntp.org)\b)|([0-3]\b(.ntp.cloud.aliyuncs.com)\b)|([0-3]\b(.us.pool.ntp.mil)\b)$</ind:pattern>
```

Comment:
shouldn't you be checking if the server specified in `var_multiple_time_servers`? 
That would allow anyone working with a tailoring file to set `var_multiple_time_servers` to the servers according to their site policy, and that might be different from this hard-coded check.

Reply:
Also isn't this entire rule the same as `chronyd_specify_remote_server`?

Reply:
I wouldn't use any regex, I would check if the `/etc/chrony.conf` file has the value set in `var_multiple_time_servers`, that way you allow users to define their own servers.

Reply:
Also @dodys 
According to your second question I will propose you to check conversation in https://github.com/ComplianceAsCode/content/pull/10278. According to me the OVAL part of the rule chronyd_specify_remote_server is not correct - because it checks for ANY word after the word server. I received a proposal to develop another rule, and because of that this PR was developed. With this PR I am trying to control better the content of /etc/chrony.conf - because the correct configuration of servers is very important for audit/ digital forensic/ how IDS/IPS are working.  
Have a nice day
Rumen